java-core/src/test/java/com/pcloud/sdk/ApiServiceTest.java

```diff
@@ -87,6 +87,12 @@ public void createFolder_ThrowsOnNullRemoteFolderArgument() throws Exception {
         instance.createFolder(null, null);
     }
 
+    @Test
```

Comment:
It would be good f the tests for the the new path-based methods also test the methods' output, not only the sanity checks presence.

Reply:
Agree. Will implement them.